modules/unpickler/src/main/scala/ch/epfl/scala/debugadapter/internal/stacktrace/Extractors.scala

```diff
@@ -20,10 +20,10 @@ object LazyInit:
 
 object LocalMethod:
   def unapply(method: binary.Method): Option[(String, Int)] =
-    if method.name.contains("$default$") then None
+    if method.name.contains("$default$") || method.name.contains("$anonfun$") || method.name.contains("proxy") then None
```

Comment:
Any `$` around proxy ? What if a user calls its local method `def proxy`?

```suggestion
    if method.name.contains("$default$") || method.name.contains("$anonfun$") || method.name.contains("proxy") then None
```